app/utils/data_seeder.rb

```diff
@@ -69,18 +79,72 @@ def load_vocabs
       { file: File.join('spec', 'fixtures', 'lae_subjects.csv'), name: "LAE Subjects",
         columns: { label: "subject", category: "category" } }
     ]
-    change_set_persister = PlumChangeSetPersister.new(
-      metadata_adapter: Valkyrie::MetadataAdapter.find(:indexing_persister),
-      storage_adapter: Valkyrie::StorageAdapter.find(:lae_storage)
-    )
-
     to_load.each do |vocab|
       change_set_persister.buffer_into_index do |buffered_change_set_persister|
         IngestVocabService.new(buffered_change_set_persister, vocab[:file], vocab[:name], vocab[:columns], logger).ingest
       end
     end
   end
 
+  def add_ephemera_fields(project)
+    [
+      ['1', 'LAE Languages'],
+      ['2', 'LAE Areas'],
+      ['3', 'LAE Areas'],
+      ['4', 'LAE Genres'],
+      ['5', 'LAE Subjects']
+    ].each do |pair|
+      # create the ephemera field
+      field_change_set = DynamicChangeSet.new(EphemeraField.new)
+      vocab = query_service.custom_queries.find_ephemera_vocabulary_by_label(label: pair[1])
+      raise "Could not ingest the field for #{pair[0]}!" unless field_change_set.validate(field_name: [pair[0]], member_of_vocabulary_id: vocab.id)
+      field_change_set.sync
+      updated_field = change_set_persister.save(change_set: field_change_set)
+      # add the field to the project
+      add_member(parent: project, member: updated_field)
+    end
+  end
+
+  def add_member(parent:, member:)
+    parent_change_set = DynamicChangeSet.new(parent)
+    parent_change_set.prepopulate!
+    parent_change_set.member_ids << member.id
+    raise "Could not update #{parent}!" unless parent_change_set.validate(member_ids: parent_change_set.member_ids)
+    parent_change_set.sync
+    updated_parent = change_set_persister.save(change_set: parent_change_set)
+    logger.info "Added #{member.class} #{member.id} to #{parent.class} #{updated_parent.title || updated_parent.box_number}"
+  end
+
+  def add_ephemera_box(project)
+    change_set = DynamicChangeSet.new(EphemeraBox.new)
+    change_set.validate(barcode: '00000000000000', box_number: '1')
+    change_set.sync
+    box = change_set_persister.save(change_set: change_set)
+    add_member(parent: project, member: box)
+    box
+  end
+
+  def add_ephemera_folders(n:, project:, box:)
+    n.times do |i|
+      change_set = DynamicChangeSet.new(EphemeraFolder.new)
+      change_set.validate(
```

Comment:
I'm getting an error when I run `rake db:seed`:

````
I, [2017-10-17T16:32:39.504438 #72631]  INFO -- : Added EphemeraBox c8d93273-47b6-4cad-ba91-73952bb34409 to EphemeraProject ["An Ephemera Project"]
rake aborted!
NoMethodError: undefined method `map' for #<Valkyrie::ID:0x007fe9075416e8>
Did you mean?  tap
/Users/escowles/src/figgy/app/change_sets/ephemera_folder_change_set.rb:114:in `language='
/Users/escowles/src/figgy/app/utils/data_seeder.rb:130:in `block in add_ephemera_folders'
/Users/escowles/src/figgy/app/utils/data_seeder.rb:128:in `times'
/Users/escowles/src/figgy/app/utils/data_seeder.rb:128:in `add_ephemera_folders'
/Users/escowles/src/figgy/app/utils/data_seeder.rb:68:in `generate_ephemera_project'
/Users/escowles/src/figgy/db/seeds.rb:14:in `<top (required)>'
/Users/escowles/.rbenv/versions/2.3.3/bin/bundle:22:in `load'
/Users/escowles/.rbenv/versions/2.3.3/bin/bundle:22:in `<main>'
Tasks: TOP => db:seed
(See full trace by running task with --trace)
```

Reply:
test failure, too. thanks. This broken when rubocop pointed out I was setting genre twice instead of language / genre each once.

Reply:
I don't know why it's doing that on language when it works fine on genre; commenting it out for now.